webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-app id="web-conferencing-admin">
-    <v-container style="width: 95%" class=" white card-border-radius pa-5">
+    <div class="white card-border-radius pa-5">
```

Comment:
please follow the example 
https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/general-settings/components/registration/HubAccess.vue
and
https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/general-settings/components/GeneralSettings.vue
You should you use <v-app>/<v-main><v-card> which will bring white color for example
px-6 instead of pa-5
overflow-hidden class ...
in addition, Is the syntax used in hubAccess with v-list is easier to read ? instead of v-col an v-row ?

I think it is preferable to follow this example